packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx

```diff
@@ -9,10 +9,17 @@ const Wrapper = styled.div`
     align-items: center;
 `;
 
+const Bg = styled.div<{ color: string }>`
```

Comment:
I think you don't need to pass props here. All styled components now have props.theme accessible by default. @slowbackspace shall correct me if I am wrong

Reply:
yess, thanks!